rpcs3/rpcs3qt/rpcn_countries.cpp

```diff
@@ -0,0 +1,82 @@
+#include "stdafx.h"
+#include "rpcn_countries.h"
+
+namespace countries
+{
+	const std::array<country_code, 72> g_countries =
+		{
+				country_code{"Japan", "jp"},
```

Comment:
Why is the indentation like this?
It should look something like:
```suggestion
	const std::array<country_code, 72> g_countries =
	{
		country_code{"Japan", "jp"},
		...
```